RHEL/7/input/xccdf/system/accounts/pam.xml

```diff
@@ -313,25 +313,25 @@ run of (<sub idref="var_password_pam_maxrepeat" /> + 1) or more identical charac
 To check the maximum value for consecutive repeating characters, run the following command:
 <pre>$ grep maxrepeat /etc/security/pwquality.conf</pre>
 Look for the value of the <tt>maxrepeat</tt> parameter. The DoD requirement is 2, which would appear as
```

Comment:
Good find. DISA's check has 2, whereas the rule title says 4. Asked them for clarification. For now, sticking to their check text (which is var_password_pam_maxclassrepeat=4, maxrepeat=2).